test/Interop/Cxx/class/Inputs/mutability-annotations.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TEST_INTEROP_CXX_CLASS_INPUTS_MUTABILITY_ANNOTATIONS_H
+#define TEST_INTEROP_CXX_CLASS_INPUTS_MUTABILITY_ANNOTATIONS_H
+
+struct HasConstMethodAnnotatedAsMutating {
+  int a;
+
+  int annotatedMutating() const __attribute__((__swift_attr__("mutating"))) {
```

Comment:
Maybe a test where there are multiple `swift_attr` attributes. Also, you can spell it like this: `struct __attribute__((swift_attr("mutating"))) X { };`.

Reply:
> Maybe a test where there are multiple `swift_attr` attributes

Good idea, I added a test.

> Also, you can spell it like this: `struct __attribute__((swift_attr("mutating"))) X { };`.

Hmm, that currently produces `mutating struct X` which doesn't look right, but it also worked this way before this change :)
Do you think `mutating` on a struct should apply it to all members methods?

Reply:
> Do you think mutating on a struct should apply it to all members methods?

Hmm, I don't know. But the test looks good how it is now. I think that's the correct way to write it.